Rudder-Bugsnag/Classes/RudderBugsnagIntegration.m

```diff
@@ -40,24 +38,21 @@ - (void) processRudderEvent: (nonnull RSMessage *) message {
     if ([type isEqualToString:@"identify"]) {
         NSDictionary *properties = message.context.traits;
         properties = [self filterProperties:properties];
-        if(properties!=nil){
+        if (properties != nil) {
             id name = [properties objectForKey:@"name"];
             id email = [properties objectForKey:@"email"];
-            [[Bugsnag configuration] setUser:message.userId withName:name andEmail:email];
+            [Bugsnag setUser:message.userId withEmail:email andName:name];
             
             for (NSString *key in properties) {
                 NSString *value = [properties objectForKey:key];
-                [Bugsnag addAttribute:key withValue:value toTabWithName:@"user"];
+                [Bugsnag addMetadata:value withKey:key toSection:@"user"];
             }
         }
-    }
-    else if ([type isEqualToString:@"screen"]){
-        [[Bugsnag configuration] setContext:message.event];
-    }
-    else if([type isEqualToString:@"track"]){
+    } else if ([type isEqualToString:@"screen"]) {
+        [Bugsnag leaveBreadcrumbWithMessage:[NSString stringWithFormat:@"Viewed %@ Screen", message.event]];
+    } else if([type isEqualToString:@"track"]) {
```

Comment:
To make the code sync with our Android integration can we check if the event name exists or not? (Refer to [this](https://github.com/rudderlabs/rudder-integration-bugsnag-android/blob/84fb64dccee8046e6ed7c4c033c85d2923ecf45c/bugsnag/src/main/java/com/rudderstack/android/integrations/bugsnag/BugSnagIntegrationFactory.java#L68) line for reference)